home/home.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./home.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
+      rel="stylesheet"
+    />
+
+    <title>FOODK</title>
+  </head>
+  <body>
+    <header>
+      <h1>Choose what you want</h1>
+    </header>
+    <main>
+      <a href="../drinks/drinks.html">
+        <section class="Drink">
+          <h2>DRINKS</h2>
+          <img src="../assets/Drinklogo.png" />
```

Comment:
There's an error in the image path
the Img name is `Drink logo.png`, you use it as `Drinklogo.png` ( the same thing with the food img)



![Screenshot from 2021-08-06 14-48-04](https://user-images.githubusercontent.com/47992412/128506059-eeac381e-1374-4de3-852b-953866da55b7.png)


_I prefer that you modify the name of the image to match what you wrote in html (without spaces in the name)_